packages/shared/src/components/FeedLayout.tsx

```diff
@@ -150,10 +151,15 @@ export default function FeedLayout({
   children,
 }: FeedLayoutProps): ReactElement {
   const { openSidebar } = useContext(SettingsContext);
+  const hiddenSidebar = useMedia(
+    [footerNavBarBreakpoint.replace('@media ', '')],
+    [true],
+    false,
+  );
 
   const currentSettings = useMedia(
     reversedBreakpoints,
-    openSidebar ? reversedSettingsSidebar : reversedSettings,
+    openSidebar && hiddenSidebar ? reversedSettingsSidebar : reversedSettings,
```

Comment:
Is this one clear enough what the different between reversedSettingsSidebar and reversedSettings is?

Reply:
I suggest extracting this one-liner to a function with a concise name.
You can define it outside the scope of the component and provide it all the details